support-models/src/main/scala/com/gu/support/workers/states/SendAcquisitionEventState.scala

```diff
@@ -10,24 +10,18 @@ import io.circe.Decoder
 import io.circe.generic.semiauto.deriveDecoder
 import org.joda.time.LocalDate
 
-trait SendAcquisitionEventState extends FailureHandlerState {
+trait SendAcquisitionEventState extends StepFunctionUserState {
```

Comment:
SendAcquisitionEvent was never a state that could go to the failure state, so this type was too limiting